trin-cli/src/main.rs

```diff
@@ -54,14 +54,22 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 fn build_request<'a>(
     method: &'a str,
-    raw_params: &'a [Box<RawValue>],
+    raw_params: &'a Option<[Box<RawValue>; 1]>,
```

Comment:
Seems like a strange/ugly type, but I'm assuming there's a non-trivial reason we use it here?

Reply:
Yeah it's not pretty, we're required to use it because of the [jsonrpc library](https://docs.rs/jsonrpc/0.12.0/jsonrpc/struct.Request.html#structfield.params) we're using. I tried a couple tweaks to clean it up, but couldn't find a suitable solution